tools/apiview/parsers/apiview-treestyle-parser-schema/main.tsp

```diff
@@ -100,4 +104,11 @@ enum CodeDiagnosticLevel {
   /** Fatal level diagnostic will block API review approval and it will show an error message to the user. Approver will have to 
   * override fatal level system comments before approving a review.*/
  Fatal:	4
-}
\ No newline at end of file
+}
+
+model NavigationItem {
```

Comment:
Will you also support custom icons?

Reply:
That can already be done Tags in this model.